README.md

```diff
@@ -9,6 +9,7 @@ This [Laravel Nova](https://nova.laravel.com) package adds a notes field to Nova
 - Ability to add notes through the UI or programmatically
 - Ability to delete user-made notes (w/ confirmation modal)
 - Customizable placeholder support
+- Set abilty to hide or show the add note input
```

Comment:
- Set abilty to hide or show the add note button

Reply:
I've changed it to:

Set ability to hide or show the 'Add Note' button